### PR TITLE
2714 Changed chevron color to gray in right component in info list item

### LINKED ISSUE #327: Chevron and non text right component default color
Describe the bug
Colour for non text right component and chevron is different than **theme.palette.text.secondary** from MUI theme.

What is the expected behaviour?
Default non-text right component and chevron colour should be **theme.palette.text.secondary** from MUI theme.

What are the steps to reproduce?
Use rightComponent props  for InfoListItem in component's storybook.
Check color CSS property

Screenshots:
Screenshot 2021-10-19 at 2 19 32 PM

### PR DESCRIPTION
**Pull request by bkarambe**
Fixes #327 

#### Changes proposed in this Pull Request:
- Updated chevron color in info list item to gray[500]

#### Screenshots:
**Before**
137876825-719cc509-18cf-4b0b-89a7-de856482e9bc

**After**
Screenshot 2022-01-10 at 5 01 20 PM

#### To Test:
- yarn start:storybook
- In InfoListItem look for the With Full Config story

### CODE REVIEW

**Review by huayunh — COMMENTED** [review]
This would cause the color to be gray500 all the time, even in the dark theme, under which the chevron shall be rendered as black200. Perhaps you could add a class for the chevron in `InfoListItem.styles.tsx`, and specify the color to be `theme.palette.text.secondary`?

File: components/src/core/InfoListItem/InfoListItem.tsx [hunk 1]
```diff
@@ -152,6 +153,7 @@ const InfoListItemRender: React.ForwardRefRenderFunction<unknown, InfoListItemPr
                     color={'inherit'}
                     role={'button'}
                     className={clsx(combine('rightComponent'), defaultClasses.flipIcon)}
+                    style={{ color: Colors.gray[500] }}
```
Comment (emclaug2) [inline components/src/core/InfoListItem/InfoListItem.tsx:156]: What's the color supposed to be in the dark theme? Will this fix work for light & dark?

Comment (daileytj) [reply components/src/core/InfoListItem/InfoListItem.tsx:156]: were using `black[200]` for dark theme... aka `theme.palette.text.secondary`

**Comment by huayunh** [comment]
remember to update readme too

File: docs/InfoListItem.md [hunk 2]
```diff
@@ -49,6 +49,7 @@ import * as Colors from '@brightlayer-ui/colors';
 | wrapInfo          | Whether to wrap info on overflow                 | `boolean`                            | no       | false          |
 | wrapSubtitle      | Whether to wrap subtitle on overflow             | `boolean`                            | no       | false          |
 | wrapTitle         | Whether to wrap title on overflow                | `boolean`                            | no       | false          |
+| chevronColor      | Color override for the chevron icon              | `string`                             | no       |                |
```
Comment (huayunh) [inline docs/InfoListItem.md:52]: You shall sort prop name alphabetically

you added a class `chevronIcon` in addition to the prop `chveronColor`. Both need to be in the readme and changelog.

**Comment by daileytj** [comment]
Functionality/Code looks good to me, just make the updates requested by Huayun and I'll approve this.

**Review by daileytj — APPROVED** [review]

**Pull request merged**